src/components/AutoSearch.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+
+import CircularProgress from '@material-ui/core/CircularProgress';
+import TextField from '@material-ui/core/TextField';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+
+import { MINIMUM_SEARCH_CHARACTERS } from '../utils/constants';
+
+interface AutoSearchProps {
+  onInputChange?: (event: object, value: string) => void;
+  searchResults: any;
+  setFieldValue: (name: string, value: any) => void;
+  formikFieldName: string;
```

Comment:
bør ikke setFieldValue og formikFieldName være optional? Ser for meg at denne kan brukes uten at man har formik også.

Reply:
godt poeng. kan rename den og gjøre formikFieldName optional

Reply:
jeg endrer til setFieldValue og fieldName - funksjonaliteten vil være lik med og uten formik og det er uansett bra å ha med name-propen selv om man ikke bruker formik. oki?

Reply:
👍